packages/ui/src/RadioButtonGroup.tsx

```diff
@@ -71,28 +70,32 @@ export function RadioButtonGroup<T extends string | number>({
                 bg={theme.colors.blue[100]}
                 boxShadow={`0 0 0 6px ${theme.colors.blue[50]}`}
                 borderRadius='100%'
-                mixBlendMode='multiply'
+                // Ensure icons fit well and are not too large
+                p={1} // Adjust padding around the icon
               >
-                <Icon as={option.icon} />
+                <Icon as={option.icon} boxSize='24px' />
```

Comment:
This change makes them larger than the design, it has them at 20.

Reply:
fixed